app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTestV2.kt

```diff
@@ -528,8 +528,26 @@ class BloodPressureEntrySheetControllerTestV2 {
   }
 
   @Test
-  fun `when BP entry is active, BP readings are valid and next arrow is pressed then date entry should be shown`() {
-    // TODO
+  @Parameters(method = "params for OpenAs types")
+  fun `when BP entry is active, BP readings are valid and next arrow is pressed then date entry should be shown`(
```

Comment:
Should this test be combined with `when save is clicked, BP entry is active and input is valid then show date entry`?

Reply:
I intentionally kept them separate to indicate that the tests treat next arrow and save as separate events even though they do the same thing right now. That said, I should name them similarly. Fixing this.